pegasus/sites.v3/code.org/views/homepage_below_hero_pd_banner.haml

```diff
@@ -0,0 +1,36 @@
+- if Homepage.show_professional_learning_banner(request)
+  = inline_css 'homepage_below_hero_pd_banner.css'
+  - header_text = 'Help students become superheroes!'
+
+  - if audience_text == 'you'
```

Comment:
Since you're always providing this value (either `you` or `educators`) this if/else could be replaced with `- text = "Our Professional Learning program helps #{audience_text} teach computer science!"`

Reply:
I tried to do this for about half an hour yesterday and couldn't get it to work, so ended up here. I can try again!

Reply:
Oh interesting! Let me know if you want to look at it together

Reply:
That would be great:)

Reply:
For anyone following... I didn't know about the double quotes. Fixed!